front/pages/api/w/[wId]/members/me/assistant_relation_override.ts

```diff
@@ -54,7 +63,7 @@ async function handler(
   }
 
   const user = auth.user();
-  if (!user) {
+  if (!user || !auth.isUser()) {
```

Comment:
Technically `!user` is 404 and `!auth.isUSer` is 401 (see other endpoints for examples)